task/common/machine/storage.go

```diff
@@ -14,23 +15,31 @@ import (
 
 	"github.com/rclone/rclone/fs"
 	"github.com/rclone/rclone/fs/sync"
+
+	"terraform-provider-iterative/task/common"
 )
 
-func Logs(ctx context.Context, remote string) ([]string, error) {
+type StatusReport struct {
+	Result string
+	Status string
+	Code   string
+}
+
+func Reports(ctx context.Context, remote, prefix string) ([]string, error) {
 	remoteFileSystem, err := fs.NewFs(ctx, remote)
 	if err != nil {
 		return nil, err
 	}
 
-	entries, err := remoteFileSystem.List(ctx, "/log")
+	entries, err := remoteFileSystem.List(ctx, "/reports")
```

Comment:
why reports vs logs?

Reply:
Logs were exclusively a line–based file with the standard output/error of the task script. Reports are a wider concept, that includes exit statuses and — soon — also a key/value dictionary with status reports from the trianing code (e.g. DVC)

Reply:
Maybe not the best word, but it's not part of the public API, so we can change it anytime we want.